content/en/apps/tutorials/application-settings.md

```diff
@@ -0,0 +1,184 @@
+
+*[Replication depth](https://docs.communityhealthtoolkit.org/apps/guides/performance/replication/#depth)* is the number of levels a specific user role is able to replicate on their application.
+
+*[A transition](https://docs.communityhealthtoolkit.org/core/overview/transitions/)* is javascript code that runs when a document is changed. A transition can edit the changed doc or do anything server side code can do for that matter.
+
+## Required Resources
+
+You should have a functioning CHT instance and have medic-conf installed locally. Read [How to set up a CHT local configuration environment](setting_up_local_configuration_environment.md)
+
+## Implementation Steps
+
+In this section, you will define a new role, set persmissions for the role, set transitions, configure a hierarchy, and upload your modified app settings file to your local environment.
+
+### 1. Set Roles and Permissions
+
+To add a new role, edit the object corresponding to `"roles"` key in `app_settings.json`. Add the new role as a key and within it, have an object with key/value pairs indicating the translation key of the role and whether it is an online or offline role.
+
+Configure a CHW role by adding the following snippet to the `"roles"` object:
+
+```json
+  "chw": {
+    "name": "usertype.chw",
+    "offline": true
+  }
+```
+
+Set permissions for the new role by adding the role to the relevant permission in the `"permissions"` object. For instance, to grant the CHW role permission to create people, add the role to the array with the key `"can_create_people"`.
+
+```json
+  "permissions": {
+    ...
+
+    "can_create_people": [
+      "program_officer",
+      "chw_supervisor",
+      "chw"
+    ],
+
+    ...
+  }
+```
+
+### 2. Set Transitions
+
+To enable or disable a transition, edit the object corresponding to the `"transitions"` key in `app_settings.json`. Enable the `transition` by setting its corresponding value to `true`, disable it by settings its value to `false`.
+
+```json
+  "transitions": {
+    "accept_patient_reports": false,
+    "conditional_alerts": false,
+    "default_responses": false,
+    "update_sent_by": false,
+    "registration": false,
+    "update_clinics": false,
+    "update_notifications": false,
+    "update_scheduled_reports": false,
+    "update_sent_forms": false,
+    "generate_patient_id_on_people": true,
+    "death_reporting": true
+  }
+```
+
+In this example, the `generate_patient_id_on_people` and `death_reporting` transitions are enabled while the rest are disabled.
+
+### 3. Set Hierarchy
+
+You can configure hierarchies by editing the object corresponding to the `"contact_types"` key in `app_settings.json`. The following code sample represents the default hierachy configuration. You can modify existing contact types by editing the objects within the array. More information about the properties of each array object can be found [here](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/hierarchy/#).
```

Comment:
would be better to use a "see also" here: https://docs.communityhealthtoolkit.org/contribute/docs/style-guide/#cross-referencing-content

Reply:
Updated.